presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveBasicTableStatistics.java

```diff
@@ -145,6 +145,45 @@ public void testCreatePartitioned()
         }
     }
 
+    @Test(groups = {HIVE_CONNECTOR, HIVE_TABLE_STATISTICS})
+    public void testAnalyze()
+    {
+        String tableName = "test_basic_statistics_analyze";
+
+        onPresto().executeQuery(format("DROP TABLE IF EXISTS %s", tableName));
```

Comment:
nit: per code style, `format` is unnecessary here, since `%s` is last in the pattern